packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -25,31 +28,31 @@ class FrameUtil {
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
    * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
+   * @param {Object} currentState - The current component state.
    * @param {boolean} useSemanticDropdown - If the dropdown should be rendered semantically instead of in a portal
    * @return {Object} - The calculated dropdown attributes.
    */
   static dropdownPosition(props, target, dropdown, maxHeight, useSemanticDropdown = false) {
     const style = ({ ...props }).style || {};
     const { height } = dropdown.getBoundingClientRect();
     const { bottom, width, top } = target.getBoundingClientRect();
+    const { height: targetHeight } = target.getBoundingClientRect();
 
     const spaceBelow = window.innerHeight - bottom;
     const maximumHeight = parseInt(style.maxHeight || maxHeight, 10) || Infinity;
-    const canFitBelow = maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
+    const canFitBelow = (maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top);
     const availableSpace = canFitBelow ? spaceBelow : top;
-
     const availableMaxHeight = Math.min(maximumHeight, availableSpace - 10);
     const isAbove = !canFitBelow;
-    const semanticTopWhenAbove = (useSemanticDropdown && isAbove
-      ? -1 * Math.min(availableMaxHeight, height)
-      : undefined
-    );
+    const semanticBottomWhenAbove = (useSemanticDropdown && isAbove ? targetHeight : undefined);
+    const semanticTopWhenAbove = (useSemanticDropdown && !isAbove ? '100%' : undefined);
```

Comment:
This evaluation is backwards and a little confusing. The semantic top is set to undefined when above and 100% when not above. 

Because the top is not dynamic, it is always 100% when appearing below, I think this logic can stay in the dropdown css. It might need a new class.

```css
.is-touch-accessible.is-below {
  top: 100%;
}
```

Reply:
Initially I had tried to have a class for top, it worked fine on initial render but on re-opening the dropdown it went below the target, which should have opened above.

Reply:
Do you know what was causing it to open below the dropdown on subsequent opening?

Reply:
After the first. run the max height of the dropdown changes which is used from the previous run resulting in the dropdown to open below.
![image](https://user-images.githubusercontent.com/42207428/81064403-47269a80-8ef7-11ea-99c3-309921e294bb.png)

Reply:
I think this could be applied in CSS with more investigation, however, the result appears to be the same. Assuming this works as intended in all supported browsers I think this is fine.